docs/images-readme.md

```diff
@@ -0,0 +1,94 @@
+# BuildKit
+
+BuildKit is a concurrent, cache-efficient, and Dockerfile-agnostic builder toolkit.
+
+Report issues at https://github.com/moby/buildkit
+
+Join `#buildkit` channel on [Docker Community Slack](http://dockr.ly/slack)
+
+# Tags
+
+### Latest stable release
+
+- [`v0.8.0`, `latest`](https://github.com/moby/buildkit/blob/v0.8.0/Dockerfile)
+
+- [`v0.8.0-rootless`, `rootless`](https://github.com/moby/buildkit/blob/v0.8.0/Dockerfile) (see [`docs/rootless.md`](https://github.com/moby/buildkit/blob/master/docs/rootless.md) for usage)
+
+### Development build from master branch
+
+- [`master`](https://github.com/moby/buildkit/blob/master/Dockerfile)
+
+- [`master-rootless`](https://github.com/moby/buildkit/blob/master/Dockerfile)
+
+
+Binary releases and changelog can be found from https://github.com/moby/buildkit/releases
+
+# Usage
+
+
+To run daemon in a container:
+
+```bash
+docker run -d --name buildkitd --privileged moby/buildkit:latest
+export BUILDKIT_HOST=docker-container://buildkitd
+buildctl build --help
+```
+
+See https://github.com/moby/buildkit#buildkit for general BuildKit usage instructions
+
+
+## Docker Buildx
+
+[Buildx](https://github.com/docker/buildx) uses the latest stable image by default. To set a custom BuildKit image version use `--driver-opt`:
+
+```bash
+docker buildx create --driver-opt image=moby/buildkit:master --use
+```
+
+
+## Rootless
+
+For Rootless deployments, see [`docs/rootless.md`](https://github.com/moby/buildkit/blob/master/docs/rootless.md)
```

Comment:
URL can be just`rootless.md`

Reply:
This `md` is used in https://hub.docker.com/r/moby/buildkit so needs to have absolute urls.